src/CMakeLists.txt

```diff
@@ -12,4 +11,7 @@ add_subdirectory(DataLib)
 
 # Add all source files
 add_executable(alara ${SOURCE_FILES})
-target_link_libraries(alara DataLib)
+
+# Set direcotry variable names
+set(DFLT_XSDIR ${xsdir})
```

Comment:
I don't think this will do what it needs to do - we should discuss (see note in Slack)